Modules/HTTP/Sources/HTTP/Message/Request.swift

```diff
@@ -64,7 +64,7 @@ extension Request {
 
         switch body {
         case let .buffer(body):
-            self.headers["Content-Length"] = body.count.description
+            self.headers["Content-Length"] = "\(body.count)"
```

Comment:
actually the stdlib recommends we use `String(describing: body.count)`. I think string interpolation is slower and I'd think `body.count.description` should be a bit faster than `String(describing: body.count)` since we're calling the property directly but that might be pure conjecture haha.